skt/kerneltree.py

```diff
@@ -274,6 +284,17 @@ def __get_remote_name(self, uri):
         return remote_name
 
     def merge_git_ref(self, uri, ref="master"):
+        """
+        Merge a git branch/reference into the tree.
+
+        Args:
+            uri: URL points to a repo containing the branch/reference to merge
+            ref: Reference to checkout, default is master.
+
+        Return:
+            A tuple (SKT_SUCCESS, reference to commit) on success merging.
```

Comment:
I'd remove the "merging" from the end (also on the line below). Or use different wording, but this one is easier :)